### PR TITLE
now user can see the error message.

### PR DESCRIPTION
**Pull request by Yelnarzh**
if user is not logged in and tries to like the event error message popup. Could be used for other errors since we don't have proper alert for error messages

### CODE REVIEW

**Review by MeiH10 — CHANGES_REQUESTED** [review]
there seem to be some error. i should be getting the message "log in to like an event", but i dont get that message the second time I like something
image

File: rpi-eventhub/frontend/src/components/EventCard/EventCard.jsx [hunk 1]
```diff
@@ -77,11 +91,13 @@ const EventCard = ({ event, isLiked, onSelect, selected }) => {
 
       if (response.status === 200) {
         const data = response.data;
+        console.log("Updating like count to", data.likes)
```
Comment (Raamzeez) [inline rpi-eventhub/frontend/src/components/EventCard/EventCard.jsx:94]: Remove console.log

File: rpi-eventhub/frontend/src/pages/Events/AllEventPage/AllEvents.jsx [hunk 2]
```diff
@@ -70,33 +74,46 @@ function AllEvents() {
...
+    useEffect(() => {
+        const fetchData = async () => {
+            setIsLoading(true);
+            await fetchEvents();
+            if(isLoggedIn) {
+                await getLikedEvents();
+            }
+            // else {
+            //     setLiked()
+            // }
```
Comment (Raamzeez) [inline rpi-eventhub/frontend/src/pages/Events/AllEventPage/AllEvents.jsx:111]: Remove comment

**Review by Raamzeez — CHANGES_REQUESTED** [review]
Remove the console log and comment

**Review by Raamzeez — APPROVED** [review]

**Review by MeiH10 — APPROVED** [review]

**Pull request merged**